pointing_config:
  distance_to_robot : 1
  alpha : 110

services:
  stand_pose: /naoqi_driver/robot_posture/go_to_posture
  say: /speech_wrapper/speak_to
  get_route_region: /semantic_route_description/get_route_region
  get_route_description: /route_description/get_region_route
  get_individual_info: /ontoloGenius/individual
  get_pointing_config: /pointing_planner/plan
  look_at: /deictic_gestures/look_at
  point_at: /deictic_gestures/point_at
  has_mesh: /uwds_ros_bridge/has_mesh
  monitor_humans: /multimodal_human_monitor/monitor_humans
  start_fact: /uwds_ros_bridge/start_fact
  end_fact: /uwds_ros_bridge/end_fact
  find_alternate_id: /multimodal_human_monitor/find_alternate_id

  activate_dialogue: /activate
  deactivate_dialogue: /deactivate

topics:
  current_facts: /base/current_facts
  coord_signals: /head_manager/head_coordination_signals

action_servers:
  move_to: m_move_to
  dialogue: dialogue_as

tuning_param:
  pointing_duration: 0.5 #seconds
  stop_tracking_dist_th: 0.3 #meters
  should_move_dist_th: 0.5 #meters
  take_robot_place_dist_th: 0.6 #meters
  lost_perception_timeout: 10 #seconds
  observe_landmark_timeout: 4 #seconds

perspective:
  world: base
  robot_place: robot_infodesk
